biz.aQute.launcher/src/aQute/launcher/Launcher.java

```diff
@@ -824,7 +837,10 @@ public void run() {
 		if (!workingdir.isDirectory())
 			throw new IllegalArgumentException("Cannot create a working dir: " + workingdir);
 
-		p.setProperty(Constants.FRAMEWORK_STORAGE, workingdir.getAbsolutePath());
+		if (System.getProperty(Constants.FRAMEWORK_STORAGE) == null)
+			p.setProperty(Constants.FRAMEWORK_STORAGE, workingdir.getAbsolutePath());
+		else
+			p.setProperty(Constants.FRAMEWORK_STORAGE, System.getProperty(Constants.FRAMEWORK_STORAGE));
```

Comment:
storage is differently and was treated wrongly in the generation of the executable JAR. The code sets the storagedir to null because the different file systems make it impossible to create an always correct path. The other values are not platform dependent. However, feel free to override the others as well.